python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -392,6 +392,13 @@ keys:
             - str
           min: 0
           max: 300
+        min_links:
+          type: int
+          description: Minimum number of ports required up before bringing up a port-channel.
+          convert_types:
+            - str
+          min: 0
+          max: 64
```

Comment:
after some research some platforms can go up to 120. Please update the max value and add a comment stating that the maximum is hardware dependent.

```
al300...10:23:13(config-s-gmulocher-if-Po42)#port-channel ?
  lacp       Set LACP parameter
  min-links  Minimum no of ports required up before bringing up a port-channel
  speed      Configure port-channel speed related parameters

al300...10:23:13(config-s-gmulocher-if-Po42)#port-channel min-links ?
  <0-120>  Minimum no of ports required up before bringing up a port-channel
```